index.js

```diff
@@ -25,17 +25,15 @@ if (/index\.covered\.js$/.test(__filename)) {
 }
 
 function NYC (opts) {
-  opts = opts || {}
-
-  this._istanbul = opts.istanbul
-  this.subprocessBin = opts.subprocessBin || path.resolve(__dirname, './bin/nyc.js')
-  this._tempDirectory = opts.tempDirectory || './.nyc_output'
-  this._reportDir = opts.reportDir
+  var config = this._loadConfig(opts || {})
 
-  var config = this._loadConfig(opts)
+  this._istanbul = config.istanbul
+  this.subprocessBin = config.subprocessBin || path.resolve(__dirname, './bin/nyc.js')
+  this._tempDirectory = config.tempDirectory || './.nyc_output'
```

Comment:
@jamestalmage I think the reasoning was that:

* if you explicitly specify the path to the bin that should be run, we assume you know the path that you want.
* otherwise we default to the bin that we know will be relative to `index.js`.